src/Controls/src/Core/Button/Button.iOS.cs

```diff
@@ -213,27 +200,40 @@ void LayoutButton(UIButton platformButton, Button button, Rect size)
 				var sharedSpacing = spacing / 2;
 
 				// These are just used to shift the image and title to center
-				// Which makes the later math easier to follow
-				imageInsets.Left += titleWidth / 2;
-				imageInsets.Right -= titleWidth / 2;
-				titleInsets.Left -= imageWidth / 2;
-				titleInsets.Right += imageWidth / 2;
+				// Which makes the later math easier to follow				
+				if (layout.Position == ButtonContentLayout.ImagePosition.Left || layout.Position == ButtonContentLayout.ImagePosition.Right)
```

Comment:
This change will impact to several tests:
<img width="352" alt="image" src="https://github.com/user-attachments/assets/8cf4fccb-1232-43d3-94ac-7644c5b885a3">

Reply:
Changes in the screenshot tests should be expected if this is working correctly so that is not necessarily a bad thing :)

Reply:
@tj-devel709 , @jsuarezruiz I have attached the button failure images also. Could you please check it?